front/lib/api/files/upsert.ts

```diff
@@ -341,6 +341,12 @@ const processingPerContentType: ProcessingPerContentType = {
     avatar: notSupportedError,
     tool_output: notSupportedError,
   },
+  "dust-application/slack": {
+    conversation: upsertDocumentToDatasource,
```

Comment:
note that it will not be upserted as we are not calling the processMethod on it (and we don't have the useCaseMetadata with the conversation ID).